docs/learn-more/index.rst

```diff
@@ -1,27 +1,34 @@
 Learn more
 ===========
 
-Here you can find the essential tools needed to run and utilize Python during your first year at DTU as a Bachelor student. 
 
-We strongly recommend first year students or those new to programming to go through the :doc:`Terminal <terminal>`
-and :doc:`IDLE <idle>` sections after installing Python. 
+.. dropdown:: Are you a first year student?
+   :color: warning
+   :icon: accessibility
 
-:doc:`Visual Studio Code <vscode>` will be introduced around week 7, and
-there's no need to rush into this section. However, we encourage you to get a head start 
-by exploring it whenever you have some spare time.
+   For first year students we recommend going through the :doc:`Terminal <terminal>` and :doc:`IDLE <idle>` sections. 
+   The :doc:`Visual Studio Code <vscode/index>` section is also very useful, but **Visual Studio Code will not be introduced until around week 7**
```

Comment:
I think we should not write an explicit week here, could you change it to `will not be introduced before later in the semester`